deployment/templates/fluentd.yaml

```diff
@@ -165,3 +167,7 @@ parameters:
   description: "Location of CA cert for validating connectiong to ElasticSearch to write cluster logs"
   name: OPS_CA
   value: "/etc/fluent/keys/ca"
+-
+  description: "JSON format configuration for log throttle management"
+  name: THROTTLE_SETTINGS
```

Comment:
Ideally we would most likely use the ConfigMap api when that's available... we could probably use a secret. My initial thought was that using a structured format would make it a) easier to parse and b) give an 'easier' way to specify x* options for y* projects